src/constants/functionKey.ts

```diff
@@ -5,6 +5,7 @@ enum FunctionKey {
   GET_ASSETS_IN_ACCOUNT = 'GET_ASSETS_IN_ACCOUNT',
   GET_VTOKEN_BALANCES_ALL = 'GET_VTOKEN_BALANCES_ALL',
   GET_HYPOTHETICAL_LIQUIDITY = 'GET_HYPOTHETICAL_LIQUIDITY',
+  BALANCE_OF = 'BALANCE_OF',
```

Comment:
This function key needs to be prefixed with `GET_` (it makes it obvious it's a query).